pkg/scheduler/plugins/elasticquota/controller.go

```diff
@@ -70,8 +70,15 @@ func NewElasticQuotaController(
 	return ctrl
 }
 
-func (ctrl *Controller) Start() {
-	go wait.Until(ctrl.Run, SyncHandlerCycle, context.TODO().Done())
+func (ctrl *Controller) Start(waitingForLeader chan struct{}) {
+	for {
+		if _, ok := <-waitingForLeader; !ok {
+			go wait.Until(ctrl.Run, SyncHandlerCycle, context.TODO().Done())
+			klog.Infof("start elasticQuota controller syncHandler")
+			break
+		}
+		time.Sleep(1 * time.Second)
```

Comment:
time.Sleep(time.Second)

Reply:
may like:

	for {
		select {
		case <-ticker.C:
			go rotateLog(matchInfoPath)
		default:
			time.Sleep(10 * time.Minute)
			klog.V(4).Infof("after sleep, begin another clean matchInfo log loop")
		}
	}

Reply:
fixed